code/sorting/src/shell_sort/README.md

```diff
@@ -1,5 +1,5 @@
 # Shellsort
-The Shellshort algorithm is a search algorithm. It's a generalization of the insertion sort that allows the exchange of items that are far apart.
+The Shellsort algorithm is a search algorithm. It's a generalization of the insertion sort that allows the exchange of items that are far apart.
```

Comment:
Could you add the title `## Explanation` here.

And also fix this doc. according to [documentation guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md)

Reply:
@iattempt  The goal was to fix a typo. Let's leave it at that. 😆